### DISCUSSION TITLE
overiding Sidebery styles

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Naezr/ShyFox#159] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/8a5e99aa-3759-4d73-81b3-299b427fd1b2)

Overiding from here dosen't work, is the only option is to go in `/home/phoenix/.mozilla/firefox/uwnvw7o9.ShyFox/chrome/ShyFox/content/shy-sidebery.css` ?

**Comment by KAGEYAM4** [comment]
if there isn't any other way than editing `chrome/ShyFox/content/shy-sidebery.css`, then is there way to live reload the changes ?

**Reply by Naezr** [reply, marked as answer]
https://reddit.com/r/FirefoxCSS/w/index/tutorials

**Reply by KAGEYAM4** [reply]
Thanks, was able to make it work. It's just there's little quirk. I have to open `Browser Toolbox`, select `MultiProcess` then use `Element picker` to select some element from `Sidebery` then go in `Style Editor` and then only `shy-sidebery.css` appears there otherwise if i don't use `Element Picker` then the `shy-sidebery.css` won't show up in `Style editor`. Weird.
